static/gsApp/views/seerAutomation/onboarding/repositorySelector.tsx

```diff
@@ -179,20 +179,28 @@ export function RepositorySelector() {
 const RepositoryRow = memo(
   ({
     repository,
+    disabled,
     checked,
     onChange,
   }: {
     checked: boolean;
+    disabled: boolean;
     onChange: (repositoryId: string, newValue: boolean) => void;
     repository: Repository;
   }) => {
     const handleChange = useCallback(() => {
```

Comment:
`handleChange` doesn't look like it does much except for the `!checked`. Could call `onChange` directly below, but that we need `!` seems weird.

Reply:
Ah yeah, we should read from the event instead

we still need `handleChange` because we want to call `onChange` with the repository.id and we can't do so in the parent because we can't memoize it properly